docs/contributing/backward-compatibility-promise.md

```diff
@@ -23,7 +23,7 @@ Once created, a git tag marking a release will never be removed or edited.
     Pre-release version format is `MAJOR.MINOR.PATCH-<alpha|beta|rc><n>`, eg. `7.0.0-beta5`.
 
 ### Current Release Plan
-Currently, `7.3` and `8.1` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
+Currently, `7.3` and `9.0` versions are supported, i.e. we are releasing patch versions for them on a regular basis.
 
 To be able to develop and improve Shopsys Framework we plan to release `MAJOR` versions almost quarterly, aiming to release a new `MAJOR` every 3-4 months.
```

Comment:
We want to amend the release cadence, currently we plan to release  major version every 12-15 months

Reply:
e05c64d5e31b74e1808c620dc55b417690b94037